src/main/java/org/isf/menu/gui/UserGroupBrowsing.java

```diff
@@ -139,6 +123,7 @@ public UserGroupBrowsing() {
 		buttonPanel.add(buttonPrivilege);
 
 		JButton buttonDelete = new JButton(MessageBundle.getMessage("angal.common.delete.btn"));
+		buttonDelete.setEnabled(false);
```

Comment:
I don't think we should disable this button, because one may need to finally hard-delete the group